classes/class3/Homework/app.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express');
+var app = express();
+var catinfo = require('./model/catinfo.js');
+var Cat = require('./model/catData').Cat;
+var exphbs = require('express-handlebars');
+var PORT = process.env.PORT || 3000;
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+app.listen(PORT, function (){
+	console.log('Running on port: ', PORT);
+});
+
+app.get('/cats/new', function (req, res){
+	var cat = new Cat({
+		name: catinfo.name[Math.floor(Math.random()*catinfo.name.length)],
```

Comment:
You probably want to separate a lot of this logic out to variables to make it cleaner